test/mpi/Makefile.mtest

```diff
@@ -37,6 +43,7 @@ $(top_builddir)/util/mtest_datatype_gen.$(OBJEXT): $(top_srcdir)/util/mtest_data
 SUMMARY_BASENAME ?= summary
 
 testing:
+	cd $(top_builddir)/dtpools && make && cd $(top_builddir)
```

Comment:
Shouldn't this be added as a dependency instead of a manual build?

Reply:
I haven't been able to force `test/mpi` to build dtpools by adding the dependency in `test/mpi/pt2pt`. I will try to fix this